### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by P0nj4**
the idea is to standardize the PR description format.

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## Name Of the PR
+
+#### Trello board reference:
```
Comment (glm4) [inline PULL_REQUEST_TEMPLATE.md]: Im not sure of including this section, considering this is a open source project and Trello is very specific to the organization. Lets wait and see what the rest of the guys think. @pMalvasio @CamilaMoscatelli @fernandatoledo @mato2593 @aguscha333 @mjurfest

**Comment by glm4** [comment]
@P0nj4 is there a magic way to use this, or just copy-paste?

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (matiasmansilla1989) [reply PULL_REQUEST_TEMPLATE.md]: Agree with @glm4. Additionally, mostly reviewers don't have access to other project's trello.

Comment (CamilaMoscatelli) [reply PULL_REQUEST_TEMPLATE.md]: Given that this is an open source project I agree that probably this section does not make much sense.

Comment (P0nj4) [reply PULL_REQUEST_TEMPLATE.md]: fixed. Trello board reference was removed

**Comment by toptierlabs** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (mato2593) [reply PULL_REQUEST_TEMPLATE.md]: Am I the only one who's still seeing the Trello section?

**Comment by MaicolBen** [comment]
@glm4 I believe it appears in the PR and you can edit it

**Comment by glm4** [comment]
@MaicolBen So magic then 🔮 . Nice!

**Comment by P0nj4** [comment]
that's right @MaicolBen

**Pull request merged**